notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/Installation.java

```diff
@@ -13,6 +13,7 @@
  */
 public class Installation implements Taggable {
     private String mPushChannel;
+    private String mPlatform = "fcm";

```

Comment:
So by default its "fcm" which will map to the Legacy HTTP API platform. Customers looking to use the new FCM V1 API platform will need to add a line of code to change this?

In a future version will we change the default behavior to be FCM V1?

Reply:
Yes, that is correct. We are planning to release a new Minor version of the SDK (and keep "fcm" as the default platform). After Google deprecates FCM Legacy, we can release a new version that defaults the platform to "fcmV1".

Alternatively, we could also release a new Major version instead, and default the platform to "fcmV1". Do you have a preference?

Reply:
I like the planned you laid out, minor version now, major version 6 months from now.